src/constants/externalResources.ts

```diff
@@ -40,13 +40,6 @@ export const getGithubCommitUrl = (
   githash: string
 ) => `https://github.com/${owner}/${repo}/commit/${githash}`;
 
-export const getLobsterTaskLink = (
```

Comment:
is it possible to delete the other lobster function in this file too? 🤔

Reply:
I left it in under the assumption that the job-logs project is gonna handle scrapping it.